openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java

```diff
@@ -867,7 +866,7 @@ public String getTypeDeclaration(Schema p) {
         Schema<?> schema = unaliasSchema(p);
         Schema<?> target = ModelUtils.isGenerateAliasAsModel() ? p : schema;
         if (ModelUtils.isArraySchema(target)) {
-            Schema<?> items = getSchemaItems((ArraySchema) schema);
+            Schema<?> items = ModelUtils.getSchemaItems(schema);
```

Comment:
why is this change necessary in the PR to update versions?

Reply:
because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0